index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Matee Safi - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+
+<body>
+      <header class="nav">
+        <nav>
+            <div class="logo-container">
+                <a class="logo-text">Matee Safi</a>
+            </div>
+            <div class="menu">
+                <img id="hamburger" src="./assets/hamburger-icon.svg" alt="hamburger menu">
+            </div>
+        </nav>
+      </header>
+    <section class="headline">
+        <div class="main-line">
+            <h1 class="orange">Hey There.<br>I'm Matee</h1>
+        </div>
+        <div class="second-line">
+            <h3 class="orange">I am a Software Developer</h3>
+        </div>
+        <div class="third-line">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-media">
+                <a href="https://github.com/matee-safi"></a><img class="icon" src="./assets/github-icon.png" alt="github">
+                <a href="https://linkedin.com/in/matee-safi-755421243)"><img class="icon" src="./assets/linkedin-icon.png" alt="linkedin"></a>
+                <a href=""><img class="icon" src="./assets/angelist-icon.png" alt="angelist"></a>
+                <a href="https://twitter.com/matee_safi"><img class="icon" src="./assets/twitter-icon.png" alt="twitter"></a></li>
+                <a href=""><img class="medium" src="./assets/medium-icon.png" alt="medium"></a>          
+        </div>
```

Comment:
:red_circle: As requested by the previous reviewer, Kindly make sure you wrap these social icons in a ul tag to group them, and li tag per icon; each icon has a link (a tag).
This code could look like this :arrow_down:  after refactoring
```
 <ul class="social-media">
     <li>
       <a href="https://github.com/matee-safi">
           <img class="icon" src="./assets/github-icon.png" alt="github">
      </a> 
    </li>
    <!-- Comment // Make sure you do the same for the remaining icons --->
 </ul>
```